app/src/main/res/layout/activity_main.xml

```diff
@@ -59,7 +75,6 @@
         android:layout_height="115dp"
         app:text="Hello"
         android:strokeColor="@color/colorPrimary"
```

Comment:
If you are making any changes or removing any attribute then specify it in the description.